x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/parser/EsqlParser.java

```diff
@@ -78,6 +82,21 @@ private <T> T invokeParser(
         }
     }
 
+    private class PostProcessor extends EsqlBaseParserBaseListener {
+        @Override
+        public void exitFunctionExpression(EsqlBaseParser.FunctionExpressionContext ctx) {
+            // TODO remove this at some point
+            EsqlBaseParser.IdentifierContext identifier = ctx.identifier();
+            if (identifier.getText().equalsIgnoreCase("is_null")) {
+                throw new ParsingException(
+                    source(ctx),
+                    "is_null function is not supported anymore, please use 'is null' or " + "'is not null' constructs instead"
+                );
+            }
+            super.exitFunctionExpression(ctx);
```

Comment:
No need to call this since it's an empty function.

Reply:
Another leftover from the default override ide implementation.